src/content/kubernetes/best-practices/index.md

```diff
@@ -35,6 +35,17 @@ You don't want your development environment to be down, because of a minor node
 
 For production usage the cluster should have at least 5 nodes and a buffer of 2 nodes.
 
+#### Master nodes
+
+Master nodes matter much more than worker nodes in order to achieve better availability and performance. Several worker nodes can be replaced at the same time with no downtime but the failure of a master node could cause a major issue if it is your only master node.
+
+The master hosts the critical components like the API, scheduler, ETCD... The load of this components will directly depend on the amount of resources they are managing, API requests and events happening in the cluster.
+
+In order to have high availability we recommend:
+
+- Cluster with 50 nodes or 1000 pods should and single master should use m5.2xlarge instance types or higher.
+- Cluster with 100 nodes or 2000 pods should use m5.4xlarge or High Availability with m5.2xlarge instances.
```

Comment:
These recommendations are specific to AWS. Since the entire page is not provider-specific, I would like to

- highlight that these recommendations are AWS-specific
- add the equivalent Azure-specific recommendations

Reply:
@giantswarm/team-celestial do you have a size recommendation for the control plane nodes?

Reply:
Standard_D4s_v3 for masters, Standard_D8s_v3 for workers